lua/lazy/help.lua

```diff
@@ -7,23 +7,30 @@ function M.index(plugin)
   if Config.options.readme.skip_if_doc_exists and vim.loop.fs_stat(plugin.dir .. "/doc") then
     return {}
   end
+  local files = vim.tbl_flatten(vim.tbl_map(function(file)
+    return vim.fn.expand(plugin.dir .. "/" .. file, false, true)
+  end, Config.options.readme.files))
   ---@type table<string,{file:string, tag:string, line:string}>
   local tags = {}
-  for _, file in ipairs(Config.options.readme.files) do
-    file = plugin.dir .. "/" .. file
+  for _, file in ipairs(files) do
     if vim.loop.fs_stat(file) then
+      local tag_filename = plugin.name
+      if file:find("lua/") then
```

Comment:
Would be better to make this more general. So build the filename based on everything after `plugin.dir`. So for example:

`nui.nvim/lua/nui/popup/README.md` -> `nui.nvim-lua-nui-popup`

Reply:
Updated.